### DISCUSSION TITLE
Previous page text is shown only partially when page name is short

### DISCUSSION THREAD

**Discussion by kamieniarz** [squidfunk/mkdocs-material#5280] [General]
As in title - I'm using polish translation and that's what I get instead of `Poprzednia strona`:
![image](https://user-images.githubusercontent.com/9063907/227917322-8885507a-03eb-466e-8da2-64a02586b47e.png)

I never had that issue with next page text. Is this a general bug or am I doing sth wrong? I'm on latest build

**Comment by squidfunk** [comment]
Looks like a bug. Please [create an issue](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) and provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by kamilkrzyskow** [comment]
Uh oh, I feel bad for this, because I saw it too after the translation, wanted to fix it myself and forgot about it, because I rarely use those buttons... 🤦 
A similar issue happens with the next button too, when making the example @kamieniarz just include sites with A, B, C titles ✌️ you'll see that the next button also isn't aligned properly.

---

Changing the `flex-grow` property of the `.md-footer__link` class seems to mostly fix it, but the general buggyness stays.
Making sure that the `Next page` text aligns with the page title from the right, and making the next button jump below the previous button like the social icons do, seems to be beyond me 😞 
<video src="https://user-images.githubusercontent.com/34622465/227946757-63ecdcbe-444c-4e4a-991e-6c6d9e6eda94.mp4">

**Comment by kamilkrzyskow** [comment]
@kamieniarz A day has passed, and I just created this issue here https://github.com/squidfunk/mkdocs-material/issues/5290 so no need to create another one. ✌️
